k8s/namespaces/jenkins/patches/cftptl/cluster-00/jenkins.yaml

```diff
@@ -176,6 +176,12 @@ spec:
                     name: ADOPTION
                     recurse: true
                     title: Adoption Data Dashboard
+                - buildMonitor:
+                    includeRegex: >-
+                      ^HMCTS_.*FPLA.*\/(fpl-ccd-configuration)\/master
```

Comment:
Do we want to include the nightly build?

Reply:
Yes, fully agreed. Personally I would include nigthly + maybe PRs? I mean we have got single repos, by average 10 PRs open so it's not much anyway to see 12 builds :). But just was quickly copying what Karthik prepared as he seem to struggle.

Reply:
Actually this will catch the nightly builds (see `.*`), however I am not sure that it should be FPLA, should it not be FPL? The links to our builds are `https://build.platform.hmcts.net/job/HMCTS_FPL/`